.github/workflows/behat-stability-without-optional.yml

```diff
@@ -0,0 +1,56 @@
+name: 'Behat stability without optional'
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches: [ main ]
+  pull_request: { }
+
+  # @todo Enable after successful testing
+  # Allows you to run this workflow manually from the Actions tab
+  # workflow_dispatch:
+
+
+# github.head_ref is only set when the workflow was triggered by a pull_request and it contains the value of the source branch of the PR.
+# github.ref_name will than only be used if the workflow was not triggered by a pull_request and it also just contains the branch name.
+env:
+  BRANCH_NAME: ${{ github.head_ref || github.ref_name }}
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "Running tests"
+  tests:
+    name: Running tests
+    runs-on: ubuntu-22.04
+    steps:
+      # Set PHP version to 8.0
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 8.0
+          coverage: none # disable xdebug, pcov
+          tools: cs2pr
+
+      # Checkout Drupal Social.
+      - uses: actions/checkout@v3
+        with:
+          repository: 'goalgorilla/drupal_social'
+          ref: '5.3.0'
+
+      # Install the version of Open Social under test.
+      - name: Composer require current branch
+        run: composer require goalgorilla/open_social:dev-$BRANCH_NAME
+
+      # Install the docker.
+      - name: Install Docker
+        run: sh scripts/social/ci/install-docker.sh
+
+      - name: Run Docker containers
+        run: docker-compose -f docker-compose-ci.yml up -d
+
+      - name: Install Open Social
+        run: docker exec -i social_ci_web bash /var/www/scripts/social/install/install_script.sh --localsettings
```

Comment:
I think this addition or test makes very impact. This is testing basic installation with stability 1. We are kind of already doing it in other tests.

Reply:
I think that this is important because it consider the tests without the optional modules. I guess this is a good one to keep